### PR TITLE
Fix: Logged-out navbar CTA parameter

### PR DESCRIPTION
**Pull request by claudiucelfilip**
#### Proposed Changes

This sets the current route as the URL parameter for the `Get Started` CTA button in the logged-out navbar

#### Testing Instructions

* Log out
* Go to `/themes` and `/plugins`
* The `Get Started` button should append the corresponding param (ie. `ref=themes-lp` and `ref=plugins-lp`)

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #

### CODE REVIEW

File: client/layout/universal-navbar-header/index.jsx [hunk 1]
```diff
@@ -200,7 +204,7 @@ const UniversalNavbarHeader = () => {
 										className="x-nav-item x-nav-item__wide"
 										titleValue={ translate( 'Get Started' ) }
 										elementContent={ translate( 'Get Started' ) }
-										urlValue="/start/business/?ref=plugins-lp"
+										urlValue={ '/start/business/?ref=' + ctaParam }
```
Comment (tyxla) [inline client/layout/universal-navbar-header/index.jsx]: Nit: could be made more readable with a template literal:

```suggestion
										urlValue={ `/start/business/?ref=${ ctaParam }` }
```

File: client/layout/universal-navbar-header/index.jsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 import WordPressWordmark from 'calypso/components/wordpress-wordmark';
 import UniversalNavbarBtnMenuItem from 'calypso/layout/universal-navbar-header/btn-menu-item.jsx';
 import UniversalNavbarLiMenuItem from 'calypso/layout/universal-navbar-header/li-menu-item.jsx';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
 
 const UniversalNavbarHeader = () => {
 	const translate = useTranslate();
 	const [ isMobileMenuOpen, setMobileMenuOpen ] = useState( false );
+	const currentRoute = useSelector( getCurrentRoute );
+	const ctaParam = currentRoute.replace( /^\//, '' ) + '-lp';
```
Comment (tyxla) [inline client/layout/universal-navbar-header/index.jsx]: This won't behave as expected if we end up loading that component for logged-in users. What's our guarantee that this won't happen?

Comment (tyxla) [reply client/layout/universal-navbar-header/index.jsx]: One way to get around it and have the expected predictable result would be to use `getSectionName()` directly and just append `-lp` to it. WDYT?

Comment (claudiucelfilip) [reply client/layout/universal-navbar-header/index.jsx]: > What's our guarantee that this won't happen?

Probably none. Using the section sounds more future-proof and allows doesn't need the `/` to be removed.
Pushed an update to address this.

**Review by tyxla — APPROVED** [review]
Looks great 🚀

**Pull request merged**